src/holdingpen/components/__tests__/__snapshots__/ExceptionsTable.test.jsx.snap

```diff
@@ -85,9 +85,8 @@ exports[`ExceptionsTable renders all exceptions on clear error search 1`] = `
   prefixCls="ant-table"
   rowClassName="exceptions-table-row"
   rowKey="recid"
-  rowSelection={null}
   showHeader={true}
-  size="large"
+  size="default"
```

Comment:
why we changed that?

Reply:
antd default changed, UI seems to be OK so that's why I didn't change anything.